.firebase/hosting.YnVpbGQ.cache

```diff
@@ -0,0 +1,44 @@
+asset-manifest.json,1542025276620,cc1d913a2c85bc35e4f94183919f8022cbaccbcbeeda4cc09ce883750e49858e
```

Comment:
Please consider what you commit into the github.
Do you think this is needed?

Never do a 
git add .
git commit -am 'message'
Unless you are very clear what you are commiting.
You should consider this: https://desktop.github.com/

Reply:
Okay I will make sure to have a clean commit next time.